client/common/cmdline.c

```diff
@@ -2937,6 +2937,20 @@ static int freerdp_client_settings_parse_command_line_arguments_int(rdpSettings*
 		{
 			settings->DisableCredentialsDelegation = !enable;
 		}
+		CommandLineSwitchCase(arg, "prevent-session-lock")
+		{
+			settings->FakeMouseMotionInterval = 180;
```

Comment:
not in core library, and with #9465 we will restrict access to `rdpSettings`. better use the `freerdp_settings_get_*` and `freerdp_settings_set_*` functions here.